src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -334,6 +334,7 @@ if (themeSwitchBtns.length) {
           "list-group-item list-group-item-action py-1"
         );
         node.setAttribute("href", `${entry.url}${currentFilePath}`);
+        node.setAttribute("role", "menuitem");
```

Comment:
@trallard here the `a` nodes in the version switcher menu (that are added by Javascript) get ARIA role `menuitem`